src/services/celoWallet.ts

```diff
@@ -42,7 +42,7 @@ async function celoWalletRequest(
         return toTxResult(kit.web3.eth.sendSignedTransaction(tx)).waitReceipt();
     } catch (e) {
         // as transaction requests get pending, they then resume all at once
-        if (e.toLowerCase().includes('known transaction')) {
+        if (e.toLowerCase().includes('failing transaction')) {
```

Comment:
did you find a "failing transaction"? It usually is a "known transaction".

Reply:
The error was showing failing instead of known when I was testing.

Reply:
:thinking:

Reply:
ahh, wait, you mean, that one that says "or always failing transaction"? No, it's a different one. Something like `"message":"known transaction: 84f89c992486ebe6e1258b12b0bbaf8ba604b88ecf0a98acc0989fa3081e8d0d"`

Reply:
ok, I'll rollback this text change.